docker-multinode/common.sh

```diff
@@ -18,7 +18,7 @@
 
 kube::multinode::main(){
   LATEST_STABLE_K8S_VERSION=$(kube::helpers::curl "https://storage.googleapis.com/kubernetes-release/release/stable.txt")
-  K8S_VERSION=${K8S_VERSION:-${LATEST_STABLE_K8S_VERSION}}
+  K8S_VERSION=v1.3.0-beta.1
```

Comment:
Please leave as-is. 
Instead, we should provide two "routes" of configuration as long as k8s supports v1.2, i.e. we may remove the current docker-bootstrap solution when v1.4 is released.